### DISCUSSION TITLE
About obtaining the macro PK stress response of each increment

### DISCUSSION THREAD

**Discussion by MaynotbeGarychan** [damask-multiphysics/DAMASK#3] [Q&A] [answered]
Hi,

I am a beginner of DAMASK. Right now I running simulation and try to obtain some results via the python processing tools by DAMASK.
There are lots of description about how to obtain the field data, but I would like to obtain the macro PK stress response of each increments, as it was printing at the terminal when running the simulation. 
Is it possible to obtain it using the python functions developed by DAMASK or any other feasible approach?

Thanks,

**Comment by MarDiehl** [comment, marked as answer]
You have to calculate the average at each step. https://damask.mpie.de/documentation/video_tutorials.html#data-analysis-using-matplotlib-and-pandas gives an example.

**Reply by MaynotbeGarychan** [reply]
Thanks for your answer. I got it.
